Modules/posixmodule.c

```diff
@@ -8490,7 +8490,12 @@ os_times_impl(PyObject *module)
     FILETIME create, exit, kernel, user;
     HANDLE hProc;
     hProc = GetCurrentProcess();
-    GetProcessTimes(hProc, &create, &exit, &kernel, &user);
+    BOOL ok;
+    ok = GetProcessTimes(hProc, &create, &exit, &kernel, &user);
+    if (!ok) {
+        PyErr_SetFromWindowsErr(0);
+        return -1;
```

Comment:
It is not even compiled.
```suggestion
        return NULL;
```